doxygen2man/doxygen2man.c

```diff
@@ -498,10 +498,12 @@ static char *allcaps(const char *name)
 	static char buffer[4096] = {'\0'};
 	size_t i;
 
-	for (i=0; i< strlen(name); i++) {
-		buffer[i] = toupper(name[i]);
+	if (name) {
+		for (i=0; i< strlen(name); i++) {
+			buffer[i] = toupper(name[i]);
+		}
+		buffer[strlen(name)] = '\0';
```

Comment:
theoretically strlen(name) could be > 4095, maybe use a variable for max(strlen(name), 4095) instead of the two strlen()'s